collections/src/main/scala/strawman/collection/immutable/WithDefaultImmutableMapOps.scala

```diff
@@ -0,0 +1,24 @@
+package strawman.collection.immutable
+
+import strawman.collection.{IterableFactoryLike, MapFactory}
+import scala.{Option}
+
+trait WithDefaultImmutableMapOps[K, +V, +C <: Map[K, V]] { self: C =>
+  
+  val underlying: C
+  
+  val defaultValue: K => V
```

Comment:
You should remove this and keep only the `default(key: K): V` method.

Reply:
I added so that I can provide the common implementation of `default(key: V): V` method from in `WithDefaultOps` trait. If I remove this, then implementors of this trait will have to implement `deafult` method in their own class with code like this `default(k) = defaultFunction(k)`. where `defaultFunction: K => V` is constructor parameter. Hence, did this change to enforce every implementor of `WithDefaultMapOps` to provide default function implementation so as to get the common implementation of `default(k: V): V` for free. This just removes small duplication from implementor's class.

Reply:
Ok then.

Reply:
You are right. Removed that `defaultValue` from `WithDefaultOps`